visp_tracker/src/visp_tracker/tracker-viewer.cpp

```diff
@@ -99,14 +99,14 @@ namespace visp_tracker
     // Compute topic and services names.
     std::string cameraPrefix;
 
-    rclcpp::rate::Rate rate (1);
+    rclcpp::Rate rate (1);
     while (cameraPrefix.empty ())
     {
       // Check for the global parameter /camera_prefix set by visp_tracker node
       if (!nodeHandle_.getParam ("camera_prefix", cameraPrefix) && !ros::param::get ("~camera_prefix", cameraPrefix))
       {
-        RCLCPP_WARN(rclcpp::get_logger("rclcpp")
-            ("the camera_prefix parameter does not exist.\n"
+        RCLCPP_WARN(rclcpp::get_logger("rclcpp"),
+            "the camera_prefix parameter does not exist.\n"
```

Comment:
A la place de `rclcpp::get_logger("rclcpp")` normalement il faut utiliser `this->rclcpp::get_logger()`
A essayer...
Si ça fonctionne, il faut le faire partout.